src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -504,6 +504,72 @@ describe('manage nsfs cli account flow', () => {
             const res = await exec_manage_cli(type, action, account_options);
             expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.InvalidAccountName.code);
         });
+
+        it('cli account add - cli create account with supplemental groups)', async function() {
+            const { type, new_buckets_path, uid, gid, name } = defaults;
+            const supplemental_groups = '303,211';
+            const expected_groups = [303, 211];
+            const account_options = { config_root, name, new_buckets_path, uid, gid, supplemental_groups};
+            const action = ACTIONS.ADD;
+            await fs_utils.create_fresh_path(new_buckets_path);
+            await fs_utils.file_must_exist(new_buckets_path);
+            await set_path_permissions_and_owner(new_buckets_path, account_options, 0o700);
```

Comment:
In all these tests we are on the `/tmp` path, right? It could be problematic with the sticky bit (as mentioned in your docs) once someone will try to use it with bucket policy?
If we can, we might want to use another `new_buckets_path` and explain why, or add a comment about it so others will understand.